x-pack/plugins/security_solution/public/common/components/header_actions/actions.tsx

```diff
@@ -272,6 +282,29 @@ const ActionsComponent: React.FC<ActionProps> = ({
             />
           </>
         )}
+        {(!securitySolutionNotesEnabled || expandableFlyoutDisabled) &&
```

Comment:
nit: with this condition (4 clauses) and the one above (3) pull them both out into variables that are named to indicate to a reader what the sum of these flags equates to.